src/pages/token-metadata/token-2022.md

```diff
@@ -364,7 +364,7 @@ These are extensions that can be enabled on mint accounts of SPL Token-2022.
 
   {% table %}
 
-  - Asset {% width="20%" %}
+  - Nft {% width="20%" %}
```

Comment:
Per offline discussion, we suggest getting more feedback on using "Nft" vs "NFT".